saleor/order/utils.py

```diff
@@ -99,7 +99,7 @@ def recalculate_order(order: Order, **kwargs):
     if order.discount:
         total -= order.discount
     order.total = total
-    order.save()
+    order.save(update_fields=["discount_amount", "total"])
```

Comment:
`total` is not a field, but multiple ones:

```suggestion
    order.save(update_fields=["discount_amount", "total_net_amount", "total_gross_amount", "currency"])
```